runtime/doc/news.txt

```diff
@@ -247,7 +247,7 @@ OPTIONS
 
 PERFORMANCE
 
-• TODO
+• Significantly reduced redraw time for long lines with syntax highlighting.
```

Comment:
"syntax" -> "treesitter"

Reply:
Are ephemeral decorations primarily used by treesitter?

Reply:
We are pretty consistent in our docs in distinguishing (legacy) "syntax highlighting" from "treesitter highlighting". So this is more about consistent user documentation than technical correctness.